### PR TITLE
Feat: Override `WebFonts`

### PR DESCRIPTION
**Pull request by eduardoformiga**
## What's the purpose of this pull request?

- [x] Test using core
- [x] Test using core with no font (default from starter)
- [x] Test using starter.store `WebFont `+ `--fs-text-face-body token` - Override
- [x] Create/Update DOCs: preview https://faststore-site-git-feat-override-webfonts-fs-854-faststore.vercel.app/docs/global-tokens/typography#add-a-custom-font

## How it works?

- It uses the `GlobalOverrides` file to get the `WebFonts.tsx` file from starter.
- if it's not present in `starter.store`, use core `WebFonts.tsx` as the default.
- use the final `WebFonts` in `_document.tsx`.

## How to test it?

Use the instructions from the starter PR

### Starters Deploy Preview

- https://github.com/vtex-sites/starter.store/pull/67

### CODE REVIEW

**Review by ArthurTriis1 — APPROVED** [review]
Great! I tried with different fonts and it worked well! hehe
Screenshot 2023-05-31 at 15 01 22

File: apps/site/pages/docs/global-tokens/typography.mdx [hunk 1]
```diff
@@ -162,4 +162,38 @@ export const mobileList = [
...
+    </>
+  );
+}
+
+export default WebFonts;
+
+```
+
+2. In your custom theme file, usually `src/themes/custom-theme.scss`, use the `--fs-text-face-body` token with the `font-family` you chose.
+Would also be good to use fallback fonts as systems fonts.
```
Comment (Mariana-Caetano) [inline apps/site/pages/docs/global-tokens/typography.mdx:187]:
```suggestion
> ℹ️ We recommend using fallback fonts as system fonts to ensure quick rendering, consistent and readable content.
```

File: apps/site/pages/docs/global-tokens/typography.mdx [hunk 2]
```diff
@@ -162,4 +162,38 @@ export const mobileList = [
...
+.theme {
+  // --------------------------------------------------------
+  // Typography (Branding Core)
+  // --------------------------------------------------------
+    --fs-text-face-body: 'Lato', -apple-system, system-ui, BlinkMacSystemFont, sans-serif;
+}
+
+```
+
+3. Run `yarn dev` and check the `font-family`.
```
Comment (Mariana-Caetano) [inline apps/site/pages/docs/global-tokens/typography.mdx:199]:
```suggestion
3. Run `yarn dev` and check the browser to see the `font-family`.
```

**Review by Mariana-Caetano — COMMENTED** [review]
LGTM ✨ I just added minor suggestions.

**Review by Mariana-Caetano — APPROVED** [review]

**Review by renatamottam — APPROVED** [review]

**Pull request merged**